ibugithub/Project-SpaceX
# Create empty nav bar

## Changes Made

- Added an images folder in the src folder.
- Modified the Navbar.js and Navbar.css file
- Add a logo to the nav
- Add new styles and remove some previous styles that you've written


## Steps to Test

1. Clone the repository and navigate to the project directory.
2. Install the project dependencies using `npm install`.
3. Start the development server using `npm start`.
4. Open the application in your web browser.
5. See The Ui is looking in accordance with the project requirements.
6. Check the console for any error messages.


## Checklist

- [ ] I have tested the changes and verified that they work as expected.
- [ ] I have added appropriate comments to the code, particularly in complex areas.
- [ ] I have reviewed the code for any potential issues or bugs.
